src/mentors.json

```diff
@@ -1324,12 +1324,12 @@
         "id": "molly_struve"
       },
       {
-        "type": "linkedin",
-        "id": "mollystruve"
+        "type": "slack",
+        "id": "DGTS6T0LA"
       },
       {
         "type": "email",
-        "id": "molly.struve@gmail.com"
+        "id": "molly.struve+mentor@gmail.com"
```

Comment:
Sorry my ignorance because I've been out these days.
Is the `+` character on purpose because is needed for something we did or is a typo? @mstruve

Reply:
It will allow me to filter mentor emails in my inbox so that I can tag and handle them accordingly. You can add '+anything' to the end of your email address and it will all go to your inbox. It's a nice trick you can use to help figure out where emails came from.

Reply:
That's cool, I did not know this! thank you 🔥